superset/migrations/shared/utils.py

```diff
@@ -185,15 +193,211 @@ def has_table(table_name: str) -> bool:
     return table_exists
 
 
-def add_column_if_not_exists(table_name: str, column: sa.Column) -> None:
+def drop_fks_for_table(table_name: str) -> None:
+    """
+    Drop all foreign key constraints for a table if it exist and the database
+    is not sqlite.
+
+    :param table_name: The table name to drop foreign key constraints for
+    """
+    connection = op.get_bind()
+    inspector = Inspector.from_engine(connection)
+
+    if isinstance(connection.dialect, SQLiteDialect):
+        return  # sqlite doesn't like constraints
+
+    if has_table(table_name):
+        foreign_keys = inspector.get_foreign_keys(table_name)
+
+        for fk in foreign_keys:
```

Comment:
Could you add log messages here in the same way you do for the other methods? Something like:

```
logger.info(f"Dropping foreign key X for table Y...")
```

Reply:
Done 
![image](https://github.com/user-attachments/assets/0aeb5fd8-9b59-4210-b8dd-ce24775ff2cc)